### DISCUSSION TITLE
Support for multiple cars

### DISCUSSION THREAD

**Discussion by jschlyter** [pypolestar/polestar_api#185] [General]
- Is it reasonable that each cars is only configurable once, or should we support multiple devices for the same car? Currently the device entry is indexed by VIN so I think not.
- Is it reasonable that you are required to input your VIN when setting up a new car? I.e., configure for username/password/VIN instead of just username/password? We can do autodiscovery if VIN is left empty, but in that case we cannot cases where two accounts has access to same car.

**Comment by leeyuentuen** [comment]
normally we can load all the car without giving the vin. i'll later check how it comes that this is broken. (see if i can use an account from someone)

**Reply by jschlyter** [reply]
I'll refactor the code so we can have multiple cars sharing the same auth without the index hack. Hope to have something to share tomorrow.

**Reply by jschlyter** [reply]
WIP in https://github.com/pypolestar/polestar_api/pull/186

**Comment by jschlyter** [comment]
https://github.com/pypolestar/polestar_api/releases/tag/1.8.0b1 now published (pre-release)
